shims/spark310/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetMaterializer.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.datasources.parquet
```

Comment:
Please put this in a package specific to this project, e.g.: `org.apache.spark.sql.execution.datasources.parquet.rapids`.  As written, this risks colliding and conflicting with the version from Spark.